wagtail/admin/views/pages.py

```diff
@@ -310,6 +311,7 @@ def edit(request, page_id):
     edit_handler = page_class.get_edit_handler()
     form_class = edit_handler.get_form_class()
 
+
```

Comment:
Please remove this extra newline, according to PEP8 we shouldn’t have two successive newlines in a function or method.